packages/charts-utils/src/format/format-by-pattern/format-by-pattern.ts

```diff
@@ -12,6 +12,7 @@ import { VARIABLE_REGEX } from './constants';
  */
 const formatByPattern = (pattern: string, value: string | number | Date) => {
   const variable = pattern.match(VARIABLE_REGEX);
+  if (value === null) return null;
```

Comment:
how about moving the early return before pattern matching?

Reply:
Then to prevent such situation we will have to add that condition everywhere where we use the formatter. I'm not sure if this is better.